src/skins/vector/css/matrix-react-sdk/views/rooms/_Autocomplete.scss

```diff
@@ -38,17 +38,29 @@
 .mx_Autocomplete_Completion_pill {
     border-radius: 17px;
     height: 34px;
+    padding: 0px 5px;
     display: flex;
     user-select: none;
     cursor: pointer;
     align-items: center;
     color: $primary-fg-color;
 }
 
-.mx_Autocomplete_Completion_pill * {
+.mx_Autocomplete_Completion_pill > * {
```

Comment:
I think it was the original intention. I think it's only worth changing if there's a child that doesn't want a margin, which for the time being there isn't :innocent: